src/client/components/ReviewEditorComponent/ReviewEditorComponent.style.css

```diff
@@ -0,0 +1,66 @@
+.container {
+  max-width: 598px;
+  background: white;
+  display: flex;
+  padding: 1.5em;
+  flex-direction: column;
+}
+.top-part {
+  display: flex;
+  align-items: center;
+  height: 4em;
+  flex-direction: row;
+}
+.profile-picture {
+  margin-right: 1em;
+  display: inline;
+  width: 40px;
+  justify-content: flex-start;
+}
+.circle-mask {
+  width: 40px;
+  height: 40px;
+  position: relative;
+  overflow: hidden;
+  border-radius: 50%;
+  margin: auto;
+}
+.circle > img {
+  display: inline;
+  margin: 0 auto;
+  max-width: 150px;
+}
+.user-photo {
+  display: inline;
+  margin: 0 auto;
+  width: 40px;
+  justify-content: flex-start;
+}
+.user-info {
+  align-items: flex-start;
+}
+.user-name {
+  font-weight: bold;
+  font-size: medium;
+  overflow: hidden;
+  min-height: 1em;
+  max-height: 2.5em;
+}
```

Comment:
Just keep
```
.user-name {
  align-items: flex-start;
  font-weight: bold;
  font-size: medium;
}
```

Reply:
In your case, since your component has more room and probably the names will not be soooo long to overflow the component, you might be ok with my suggestion